.travis/install.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [[ $SODIUM_INSTALL == 'system' ]]; then
+    wget https://download.libsodium.org/libsodium/releases/LATEST.tar.gz
+    tar zxvf LATEST.tar.gz
+    cd libsodium-*
+    ./configure
+    make
+    make check
+    sudo make install
+    sudo ldconfig
+fi
+
+if [[ "${TOXENV}" == "pypy" ]]; then
+    git clone https://github.com/yyuu/pyenv.git ~/.pyenv
```

Comment:
I think you might need to replace this with a `git pull` in `~/.pyenv`, the dir already exists.